ls1intum/Artemis
# Disable SGI drag and drop in readOnly mode

### This PR is a collaboration together with @TobiasPr (Apollon)
### Checklist

### Motivation and Context


We want to disable drag and drop of the structured grading instructions if the view is meant for displaying the grading instructions only and not for assessment.
### Description

passed the readOnly property to the SGI layout component. The draggable attribute is then set according to the readOnly value.
### Steps for Testing


make sure the drag and drop for all 3 exercise types (modeling-, file-upload- and text-exercise) is
**enabled** in: 
- the assessment view

**disabled** in:
 - the exercise detail view
- the tutor/exercise dashboard
- the example submission/assessment view

### Screenshots
